recipes-support/camera-streaming-daemon/camera-streaming-daemon_0.1.bb

```diff
@@ -3,7 +3,7 @@ DEPENDS = "avahi gstreamer1.0 gstreamer1.0-rtsp-server glib-2.0 python python-fu
 LICENSE = "Apache-2.0"
 LIC_FILES_CHKSUM = "file://LICENSE;md5=93888867ace35ffec2c845ea90b2e16b"
 
-SRCREV = "a6dae306ac319b5b3cd141e1aac8b5e43c0f965e"
+SRCREV = "7d12c0680cf6101d9d4cf309c3a589ca87eefa93"
```

Comment:
It worth updating to 47ae9dde6380493dcba0794bfef1962c6744f15c (which fixes a memory leak)